test/modules/api.spec.js

```diff
@@ -6,15 +6,18 @@ describe('api', () => {
   describe('#createSamples ', () => {
     beforeEach(() => {
       boxBarcode = 'aBoxBarcode'
-      plateBarcodes = ['aBarcode1', 'aBarcode2']
+      labwhereResponse = {
+        success: true,
+        plateBarcodes: ['aBarcode1', 'aBarcode2']
```

Comment:
think this should probably be called `data: ['aBarcode1', 'aBarcode2']` to be consistent with `sequencescape.js`?

Reply:
My next move was to change sequencescape.js. Data is not descriptive. To me plateBarcodes makes much more sense.

Reply:
Hmm, I think I disagree, because it would be nice to have a consistent object structure returned from these functions. So, it always returns `success: true/false`, it could also always return `data: <some data here>`, but the nature of the data would be different for different method calls.

I don't feel strongly about it, but that's what I was imagining.

Reply:
Happy to discuss further at some point. Always ready to have a good debate! I'm not saying my way is correct just trying different options to see what works. Data feels like adding an extra layer of complexity. For example what we were doing before was creating data and then adding plateBarcodes and then at the other end pulling out data and plateBarcodes. This way it is always plateBarcodes all the way through. No extra code needed.